src/backend/api/Fusion.Resources.Domain/Commands/Tasks/TaskSource.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Fusion.Resources.Domain
```

Comment:
I would prefer to prefix enums with Query. (Not sure if all other domain models follow the same naming)